raiden/api/objects.py

```diff
@@ -0,0 +1,112 @@
+
+class FlatList(list):
+    """
+    This class inherits from list and has the same interface as a list-type.
+    However, there is a 'data'-attribute introduced, that is required for the encoding of the list!
+    The fields of the encoding-Schema must match the fields of the Object to be encoded!
+    """
+
+    @property
+    def data(self):
+        return list(self)
+
+    def __repr__(self):
+        return '<{}: {}>'.format(self.__class__.__name__, list(self))
+
+
+class ChannelList(FlatList):
+    pass
+
+
+class TokensList(FlatList):
+    pass
+
+
+class AddressList(FlatList):
+    pass
+
+
+class PartnersPerTokenList(FlatList):
+    pass
+
+
+class EventsList(FlatList):
+    pass
+
+
+class Address(object):
+    def __init__(self, token_address):
+        self.address = token_address
+
+
+class PartnersPerToken(object):
+    def __init__(self, partner_address, channel):
+        self.partner_address = partner_address
```

Comment:
This class indicates the counterpart of a channel. Mimics the STACHAP spec with the PRTY command:

```
# states that Marcelo's address will be authorized to join,
PRTY 0x5BF81fD7BDFD84465803300De692d8063a22d736
```